docs/billing-integration.md

```diff
@@ -94,9 +97,35 @@ data:
       # The aggregation section indicates that reports that the agent
       # receives for this metric should be aggregated for a specified
       # period of time prior to being sent to the reporting endpoint.
+      # This is independent from how often the app reports usage to
```

Comment:
I'm confused, if the agent defers reporting until the metric has been aggregated for that period of time, how is it independent from how often the agent reports usage?

Reply:
The `aggregation.bufferSeconds` is indeed tied to how often the *agent* reports usage - it defines it exactly. However, this is independent from how often the *app* reports usage to the agent.

Do you think it's better to leave off? The two kinds of reporting (app -> agent, agent -> google) didn't seem very clearly distinguished to partners, so I was trying to speak to that here.

Reply:
Got it, didn't understand the distinction being made between *app* and *agent* reporting. Up to you for what you think is best

Maybe change paragraph, "Metrics reported by the application are aggregated by the agent for a specified period of time and sent to the reporting endpoint (e.g. Google Service Control)"

Reply:
Thanks - updated to the suggested paragraph.